tests/components/denonavr/test_config_flow.py

```diff
@@ -439,6 +444,7 @@ async def test_options_flow(hass):
         CONF_ZONE2: True,
         CONF_ZONE3: True,
         CONF_UPDATE_AUDYSSEY: False,
+        CONF_USE_TELNET: False,
     }
 
```

Comment:
We are missing a test that validates that `CONF_USER_TELNET` can be disabled

Reply:
Updated test in https://github.com/home-assistant/core/pull/85980/commits/5e3888c2cdeec78d75d5fe4dee7eee080b061de1